app/metrics.yaml

```diff
@@ -497,6 +497,43 @@ metrics:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+  recently_used_pwa_count:
+    type: counter
+    lifetime: application
+    description: |
```

Comment:
Most of this can be moved to documentation. Keep this description as succinct as possible

Reply:
This is the documentation though, e.g. `metrics.md` is directly generated from this file.